packages/land-sale/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@sandbox-smart-contracts/land-sale",
+  "version": "0.0.1",
+  "description": "Land Sale Contract",
+  "files": [
+    "contracts"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "lint": "eslint --max-warnings 0 \"**/*.{js,ts}\" && solhint --max-warnings 0 \"contracts/**/*.sol\"",
+    "lint:fix": "eslint --fix \"**/*.{js,ts}\" && solhint --fix \"contracts/**/*.sol\"",
+    "format": "prettier --check \"**/*.{ts,js,sol,md}\"",
+    "format:fix": "prettier --write \"**/*.{ts,js,sol,md}\"",
+    "test": "hardhat test",
+    "coverage": "hardhat coverage --testfiles 'test/*.ts''test/*.js'",
+    "hardhat": "hardhat",
+    "compile": "hardhat compile",
+    "size": "hardhat size-contracts",
+    "layout": "hardhat clean && hardhat compile && hardhat check",
+    "release": "release-it"
+  },
+  "mocha": {
+    "require": "hardhat/register",
+    "timeout": 40000,
+    "_": [
+      "test/**/*.ts"
+    ]
+  },
+  "devDependencies": {
+    "@ethersproject/abi": "^5.7.0",
+    "@ethersproject/providers": "^5.7.2",
+    "@nomicfoundation/hardhat-chai-matchers": "^2.0.2",
+    "@nomicfoundation/hardhat-ethers": "^3.0.5",
+    "@nomicfoundation/hardhat-network-helpers": "^1.0.10",
+    "@nomicfoundation/hardhat-toolbox": "^4.0.0",
+    "@nomicfoundation/hardhat-verify": "^2.0.2",
+    "@nomiclabs/hardhat-etherscan": "^3.1.7",
+    "@openzeppelin/contracts-upgradeable": "^4.9.5",
```

Comment:
Should be in dependencies, not dev deps

Reply:
Good call, this would only surface when we tried to import this package somewhere else. Thanks!